### PR TITLE
ac/1.8.7.1-updates-time-series-recipe-tutorial-retail-sales-forecasting

### PR DESCRIPTION
**Pull request by Castro38**
**Updated the 1.8.7.1 DAI Lab 9 in Aquarium:**
- Images (blurred out all DAI versions)
- Content to match images
- Tested links, fixed broken links
- Added to the concepts section
- Added an Appendix B with Time Series FAQ

### CODE REVIEW

File: Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md [hunk 1]
```diff
@@ -765,13 +776,13 @@ One important thing to note is why we changed the **Scorer**  that Driverless AI
...
+When looking at both varialbe importance results, we can see that for the pre-ran experiment the variable importance that had the most importance was `106 EWMA Lag` or the Exponentially Weighted Moving Average, which calculates the exponentially moving average of a target or feature lag, compared to the lag of 52 weeks for the new experiment. The feature that we see in the pre-ran experiment is a weighted moving average of what happened in various weeks over a course of 2 years; this is a more complex feature than the 52 weeks lag, and that is expected because we built a more complex model from the pre-ran experiment. Although the 52 weeks lag would help make the prediction for a peak value more accurate, our more complex model is trained to be able to predict any point in time, compared to our simple model which would make predictions based on the 1 year lag. Note that the 52 lag is indeed, one of the important variables in the complex model, but is not the most important one.
```
Comment (5675sp) [inline Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md:785]: change from: "When looking at both varialbe importance"

to: "When looking at both variable importance"

File: Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md [hunk 2]
```diff
@@ -820,18 +831,101 @@ Where you will learn:
...
+
+For instance: Let’s assume we have train = [1,2,3,4,5,6,7,8,9,10] and test = [12,13], where integers define time periods (e.g., weeks). For this example, the most natural train/valid split that mimics the test scenario would be: train = [1,2,3,4,5,6,7] and valid = [9,10], and month 8 is not included in the training set to allow for a gap. Note that we will look at the start time and the duration of the test set only (if provided), and not at the contents of the test data (neither features nor target). If the user provides validation = [8,9,10] instead of test data, then this could lead to inferior validation strategy and worse generalization. Hence, we use the user-given test set only to create the optimal internal train/validation splits. If no test set is provided, the user can provide the length of the test set (in periods), the length of the train/test gap (in periods) and the length of the period itself (in seconds).
```
Comment (5675sp) [inline Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md:869]: remove the ":" in the following sentence: " test scenario would be: train = "

File: Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md [hunk 3]
```diff
@@ -820,18 +831,101 @@ Where you will learn:
...
+
+**Why does the gap between train and test matter? Is it because of creating the lag features on the test set?**
+
+Taking the gap into account is necessary in order to avoid too optimistic estimates of the true error and to avoid creating history-based features like lags for the training and validation data (which cannot be created for the test data due to the missing information).
+
+**In regards to applying the target lags to different subsets of the time group columns, are you saying Driverless AI perform auto-correlation at “levels” of the time series? For example, consider the Walmart dataset where I have Store and Dept (and my target is Weekly Sales). Are you saying that Driverless AI checks for auto-correlation in Weekly Sales based on just Store, just Dept, and both Store and Dept?**
```
Comment (5675sp) [inline Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md:875]: - Change the following from "In regard to applying" to "In regard to applying"

- Change the following word from  "auto-correlation" to "autocorrelation"

File: Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md [hunk 4]
```diff
@@ -690,34 +701,34 @@ At the end of the experiment, a similar page will appear:
 10\. Go to the top of the page and:
 
 1. Select **Add Panel**
-2. On the new panel, click on the **Select a model interpretation**, then select the Time Series Model named : ``Time Series Forecasting: karitiwi``. This will bring in the pre-ran experiment’s MLI results. Click on **Show Summary** for both experiments to compare experiment settings:
+2. On the new panel, click on the **Select a model interpretation**, then select the Time Series Model named : ``Time Series Forecasting: walirivi``. This will bring in the pre-ran experiment’s MLI results. Click on **Show Summary** for both experiments to compare experiment settings:
```
Comment (5675sp) [inline Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md:704]: put the word "name" and ":" together

File: Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md [hunk 5]
```diff
@@ -733,15 +744,15 @@ This experiment was run in another environment with similar parameters except fo
 
 The above settings are recommended settings for time series problems, notice the high accuracy, time, and lower interpretability compared to the settings from task 2. Time-series experiments are very special cases. As a result, it is highly encouraged that the experiments are run with the default settings given by Driverless AI.
 
-For a time-series experiment an **Accuracy** of 10 is highly encouraged because it forces many time splits (time splits are critical for stability and prevents overfitting) and allows for multiple window validation. If you must run a time-series experiment with anything lower than a 10, the lowest recommended setting for accuracy is a 5.
+For a time-series experiment the Driverless AI  **Accuracy** default value is highly encouraged because it forces many time splits (time splits are critical for stability and prevents overfitting) and allows for multiple window validation. If you must run a time-series experiment with anything lower than the default, the lowest recommended setting for accuracy is a 5.
```
Comment (5675sp) [inline Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md:747]: Close the extra space  in between "AI Accuracy"

File: Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md [hunk 6]
```diff
@@ -27,7 +28,6 @@ Driverless AI has its own recipes for time-series forecasting that combines adva
 You will need the following to be able to do this tutorial:
 
 - Basic knowledge of Machine Learning and Statistics
-- A Driverless AI environment
 - Basic knowledge of Driverless AI or doing the [Automatic Machine Learning Introduction with Drivereless AI Test Drive](https://h2oai.github.io/tutorials/automatic-ml-intro-test-drive-tutorial/#0)
```
[the remark this answers is not in the document]
Comment (Castro38) [reply Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md:31]: Done - Thanks

File: Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md [hunk 7]
```diff
@@ -53,11 +53,9 @@ The train.csv has the store number, department, date, weekly sales, and whether
...
 
-The training dataset in this tutorial contains 73,165 rows and a total of 11 features (columns) and is about 5 MB. The test dataset contains about 16,000 rows and a total of 11 features (columns) and is about 1 MB.
-
 ### Datasets Overview
 
-If you are using Aquarium as your environment, then the following labs, **Test Drive** and **Introduction to Driverless AI**, will have this tutorial training and test subsets of the Retail Store Forecasting dataset preloaded for you. The datasets will be located on the **Datasets Overview** page. You will also see two extra data sets, which you can ignore for now as they are used for another tutorial. 
+If you are using Aquarium as your environment, then the lab **Driverless AI Test Drive (x.x.x.x.)(LTS)** will have this tutorial training and test subsets of the Retail Store Forecasting dataset preloaded for you. The datasets will be located on the **Datasets Overview** page. You will also see two extra data sets, which you can ignore for now as they are used for another tutorial. 
 
 **NOTE:** To learn how to add the two datasets from the Driverless AI file system see [Appendix A: Add the Datasets](#task-7-appendix-a-add-the-datasets).
```
[the remark this answers is not in the document]
Comment (Castro38) [reply Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md:60]: Done -thanks for the feedback

File: Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md (hunk 1 above)
Comment (Castro38) [reply Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md:785]: Done -thanks

**Pull request merged**